markdown/blog/community-spotlight-ashmit.md

```diff
@@ -0,0 +1,136 @@
+
+***Ashmit:** I have two main pieces of advice: be patient and work hard. Open source is community-driven, and maintainers often have full-time jobs, so responses to contributions can take time. Patience is essential. Hard work is equally important—keep learning, refining your skills, and making meaningful contributions.*
+
+## Looking Ahead: What’s Next for Ashmit?
+
+**Bami: What are your plans after GSoC? Will you continue contributing to AsyncAPI or explore other open-source projects?**
+
+***Ashmit:** Right now, I am focusing on completing my degree and securing a position in a good software company. However, I will continue contributing to open source because it has given me so much, and I feel a responsibility to give back. While AsyncAPI remains my primary focus, I am also interested in projects like Chromium, though I know it’s quite a challenging one to take on.*
+
+**Bami: Are there any new features or improvements you’d like to see in AsyncAPI?**
+
+***Ashmit:** Specifically for the conference website, I have some ideas for future improvements. However, in general, AsyncAPI is constantly evolving, and I look forward to contributing to its growth and helping the community in any way I can.*
+
+
+To add a bit of fun to our spotlight interview, we played a **"This or That"* game with Ashmit. 
+
+**Bami: Okay, so I have a fun activity to do with you. It is called this for that. So Github or Gitlab.**
+
+***Ashmit:** Github*
+
+**Bami: Slack or discord.**
+
+***Ashmit:** Discord*
+
+
+**Bami: LinkedIn or Twitter?**
+
+
+***Ashmit:** Neither. I deleted both of them just this week. Yeah, both are toxic in their own ways to be honest.*
+
+**Bami: Google Docs or Microsoft Word.**
+
+***Ashmit:** Google Docs.*
+
+**Bami VS code or X code**
+
+***Ashmit:** Vs code*
+
+**Bami: Window or Linux**
+
+***Ashmit:** Linux. Not hardcore Linux like fedora though.* 
+
+**Bami: Apple or Samsung.**
+
+***Ashmit:** Samsung*
+
+**Bami: How do you like to spend your time outside coding? Do you have any hobby or interest?**
+
+***Ashmit:** For hobbies, I might say cooking is one of them. Yeah I do like to cook sometimes because if I have some free time that's where I go towards the kitchen.* 
+
+**Bami: Is there anyone or a lot of people you would like to thank or give shout out to from the community or the GSoC program, either your other mentees, your mentors or other community members.**
+
+***Ashmit:** My mentors are the ones who I would like to give shout out to like is [Maya](https://www.linkedin.com/in/aishatmuibudeen/), [Thule](https://www.linkedin.com/in/v-thulisile-sibanda/), and [Ace](https://www.linkedin.com/in/acebuild/). And on top of that, I would like to thank [Lukasz](https://www.linkedin.com/in/lukasz-gornicki-a621914/) as well because he has been quite helpful. There were some issues in my project that required his Assistance throughout and he was there for it. Honestly, that's quite very nice of him because he doesn't have to do it.*
```

Comment:
```suggestion
***Ashmit:** My mentors are the ones who I would like to shout out to like is [Maya](https://www.linkedin.com/in/aishatmuibudeen/), [Thule](https://www.linkedin.com/in/v-thulisile-sibanda/), and [Ace](https://www.linkedin.com/in/acebuild/). And on top of that, I would like to thank [Lukasz](https://www.linkedin.com/in/lukasz-gornicki-a621914/) for being quite helpful. Some issues in my project required his assistance, and he was there for them. That's nice of him because he didn't have to do it.*
```